core/src/avm2/globals/flash/display/bitmap_data.rs

```diff
@@ -463,6 +463,73 @@ pub fn set_pixels<'gc>(
     Ok(Value::Undefined)
 }
 
+/// Implements `BitmapData.setPixels`.
+pub fn set_vector<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let rectangle = args.get_object(activation, 0, "rect")?;
+    let vec = args.get_object(activation, 1, "inputVector")?;
```

Comment:
Looking at the previous comment this should be `imputVector` instead of `inputVector`, right? It might be good to add a "sic" there.